src/config/dynamic-theme-fixes.config

```diff
@@ -2289,6 +2290,22 @@ INVERT
 
 ================================
 
+nextdns.io
+
+CSS
+.index .hero .content .logo {
+    background-image: url(/static/images/logo.png);
+    background: 181a1b;
```

Comment:
Use `${Color to be darken}` so `${white}` become `rgb(23,23,23)` -> Dark color
and use # Before it

Reply:
That makes the logo invisible.

Reply:
My bad, but it doesn't change anything from my fix.

Reply:
Okay